migrations/920-adds-websitesubmission-model-back.sql

```diff
@@ -0,0 +1,38 @@
+CREATE TABLE `websites_websitesubmission_keywords` (
+    `id` integer AUTO_INCREMENT NOT NULL PRIMARY KEY,
+    `website_id` integer UNIQUE,
+    `tag_id` integer unsigned NOT NULL,
+    UNIQUE (`website_id`, `tag_id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+ALTER TABLE `websites_websitesubmission_keywords` ADD CONSTRAINT `websites_websitesubmission_keywords_tag_id` FOREIGN KEY (`tag_id`) REFERENCES `tags` (`id`);
+
+CREATE TABLE `websites_websitesubmission` (
+    `id` integer AUTO_INCREMENT NOT NULL PRIMARY KEY,
+    `created` datetime NOT NULL,
+    `modified` datetime NOT NULL,
+    `date_approved` datetime NULL,
+    `name` integer unsigned UNIQUE,
+    `description` integer unsigned UNIQUE,
+    `categories` longtext NOT NULL,
+    `detected_icon` varchar(255) NOT NULL,
+    `icon_type` varchar(25) NOT NULL,
+    `icon_hash` varchar(8) NOT NULL,
+    `url` varchar(255) NOT NULL,
+    `canonical_url` varchar(255),
+    `works_well` integer unsigned UNIQUE,
+    `submitter_id` integer unsigned UNIQUE,
+    `public_credit` bool NOT NULL DEFAULT FALSE,
+    `why_relevant` longtext NOT NULL,
+    `preferred_regions` longtext NOT NULL,
+    `approved` bool NOT NULL DEFAULT FALSE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE INDEX websites_websitesubmission_created_idx ON websites_websitesubmission (created);
```

Comment:
I was thinking only `modified` since it's part of the `ordering` in the model. If we will query it with a `created` order often this is fine.